### DISCUSSION TITLE
Styles

### DISCUSSION THREAD

**Discussion by zilinjak** [carltongibson/django-sphinx-view#18] [General]
Hello, 
Btw this tool looks awesome keep that up.

I was just wondering if there is and option how to make the website look bit more nicer with like template, or default css provided to builder.

**Comment by carltongibson** [comment]
Hi 👋

Your base template can include any styles you wish. (You have an HTML page in which to include CSS.) 

That's not something I want to ship in the package, but there are plenty of themes out there. 

I'd suggest checking out Sphinx Themes: https://sphinx-themes.org

These target the Sphinx HTML builder, but you could extract the stylesheets, and use them as a foundation — the core HTML of the page is identical (since it's Sphinx generated) — the differences being only in the HTML skeleton of your page. 

I hope that helps.
